html/index.html

```diff
@@ -0,0 +1,480 @@
+                    <img src="../assets/Bars/Status Bars/Wifi.png" alt="">
+                    <img src="../assets/Bars/Status Bars/Battery.png" alt="">
+    
+                </div>
+    
+            </div>
+    
+            <div class="header">
+    
+    
+            
+    
+                <div class="left-text">My Logo</div>
+    
+                <div class="right-hamburger">
+    
+                    <img src="../assets/Union.png" alt="">
+    
+                </div>
+    
+            </div>
+    
+            <div class="top-body-background">
+                <div class="top-body">
+    
+                    <h2>I'm Toyyib, Glad to see you!</h2>
+            
+                    <p>
+                    I’m a software developer! I can help you build a product , feature or
+                    website Look through some of my work and experience! If you like what
+                    you see and have a project you need coded, don’t hestiate to contact
+                    me.
+                    </p>
+            
+                </div>
+            
+                <div class="footer-connect">
+                    
+                    <p><a href="#">LET'S CONNECT</a> </p>
+            
+                </div>
+            
+                <div class="top-footer">
+            
+                    <img src="../assets/twitter.svg" alt="">
+                    <img src="../assets/linkdn.svg" alt="">
+                    <img src="../assets/medium.svg" alt="">
+                    <img src="../assets/github.svg" alt="">
+                    <img src="../assets/rabbit.svg" alt="">
+            
+                </div>
+    
+            </div>
+    
+        </div>
+    
+        
+        <div class="second-container">
+    
+            <div class="inner-second-container">
+    
+                <div class="top-image">
+    
+                    <img src="../assets/Snapshoot Portfolio.png" alt="">
+    
+                </div>
+    
+                <div class="text-inner-container">
+    
+                    <h2> Tonic </h2>
+    
+                    <div class="canopy-inner-container">
+    
+                        <p id="text1">CANOPY</p>
+    
+                        <ul>
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
+    
+                    </div>
+    
+                    <div class="canopy-daily">
+    
+                        <p>
+                            A daily selection of privately 
+                            personalized reads; no accounts 
+                            or sign-ups required.
+        
+                        </p>
+    
+                    </div>
+    
+    
+                    <div class="canopy-list">
+                        <p>html</p>
+                        <p>css</p>
+                        <p>javaScript</p>
+                    </div>
```

Comment:
- [ ] Please make use of `ul, li` tags for all technologies in all your project cards as per the requirement.